core/SCsub

```diff
@@ -15,20 +15,19 @@ txt = "0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0, 0,0,0,0,0,0,0,0"
 if ("SCRIPT_AES256_ENCRYPTION_KEY" in os.environ):
     e = os.environ["SCRIPT_AES256_ENCRYPTION_KEY"]
     txt = ""
-    ec_valid = True
-    if (len(e) != 64):
-        ec_valid = False
-    else:
+    ec_valid = (len(e) == 64)
+    if (ec_valid):
 
         for i in range(len(e) >> 1):
-            if (i > 0):
-                txt += ","
             txts = "0x" + e[i * 2:i * 2 + 2]
             try:
                 int(txts, 16)
             except:
                 ec_valid = False
+            if(i > 0):
```

Comment:
There's a missing space after `if`. This should be moved back up to where it was, there's no reason to introduce a change in the `git blame` if it's not changing anything. Commits should then be squashed into one.